reference/forms/types/percent.rst

```diff
@@ -50,6 +51,15 @@ scale
 By default, the input numbers are rounded. To allow for more decimal places,
 use this option.
 
+symbol
+~~~~~~
+
+**type**: ``boolean`` **default**: ``true``
+
+By default, fields are rendered with a percentage sign "`%`" after the input
+box. Setting this value to ``false`` will disable the percentage sign from
```

Comment:
´´´diff
- will disable the percentage sign from being displayed.
+ will not show the percentage sign.
```

what do you think? Much simpler to read/understand